lib/rubocop/cop/style/multiline_when_then.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks uses of the `then` keyword
+      # in multi-line when statements.
+      #
+      # @example
+      #   # bad
+      #   case str
+      #   when 'rubocop' then
+      #     puts 'rubocop'
+      #   else
+      #     puts 'not rubocop'
+      #   end
+      #
+      #   # good
+      #   case str
+      #   when 'rubocop'
+      #     puts 'rubocop'
+      #   else
+      #     puts 'not rubocop'
+      #   end
+      #
+      #   # good
+      #   case str
+      #   when 'rubocop' then puts 'rubocop'
+      #   else
+      #     puts 'not rubocop'
+      #   end
+      #
+      class MultilineWhenThen < Cop
+        include RangeHelp
+
+        MSG = 'Do not use then for multiline when statement.'
```

Comment:
Please, surround `then` and `when` in backticks.

Reply:
OK, I'll fix it soon.